exporter/prometheusremotewriteexporter/exporter_concurrency_test.go

```diff
@@ -137,15 +134,22 @@ func Test_PushMetricsConcurrent(t *testing.T) {
 		resp, checkRequestErr := http.Get(server.URL)
 		require.NoError(c, checkRequestErr)
 		assert.NoError(c, resp.Body.Close())
-	}, 5*time.Second, 100*time.Millisecond)
+	}, 15*time.Second, 100*time.Millisecond)
 
 	var wg sync.WaitGroup
 	wg.Add(n)
+	maxConcurrentGoroutines := runtime.NumCPU() * 4
```

Comment:
If there are still failures on this test it may be necessary to reduce even further the amount of concurrency allowed here.